clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -558,6 +558,7 @@ static void CheckForIncompatibleSYCLLoopAttributes(
                                                                          Attrs);
   CheckForDuplicationSYCLLoopAttribute<SYCLIntelFPGASpeculatedIterationsAttr>(
       S, Attrs);
+  CheckForDuplicationSYCLLoopAttribute<SYCLIntelFPGALoopCountAvgAttr>(S, Attrs);
```

Comment:
We should find out from the attribute authors why they want this sort of check. FWIW, C++ and C both removed such requirements for standard attributes because this sort of check turned out to be harmful in practice.

(I'm fine leaving the check in this patch, my comment applies generally to all of our custom attributes.)